dashboard/src/routes/issue/$issueId/version/$versionNumber/route.tsx

```diff
@@ -0,0 +1,32 @@
+import { z } from 'zod';
+
+import { createFileRoute, stripSearchParams } from '@tanstack/react-router';
+
+import {
+  defaultValidadorValues,
+  zTableFilterInfoDefault,
+  zTableFilterInfoValidator,
+} from '@/types/tree/TreeDetails';
+import { DEFAULT_DIFF_FILTER, DEFAULT_ORIGIN } from '@/types/general';
+import { DEFAULT_TIME_SEARCH } from '@/pages/treeConstants';
+
+const defaultValues = {
+  origin: DEFAULT_ORIGIN,
+  tableFilter: zTableFilterInfoDefault,
+  diffFilter: DEFAULT_DIFF_FILTER,
+  currentPageTab: defaultValidadorValues.tab,
+  intervalInDays: DEFAULT_TIME_SEARCH,
+  treeSearch: '',
+  hardwareSearch: '',
+  treeIndexes: [],
+  treeCommits: {},
+};
+
+const issueDetailsSearchSchema = z.object({
+  tableFilter: zTableFilterInfoValidator,
```

Comment:
If IssuesDetails page does not use tableFilter directly you can keep this schema as an empty zod object. But you can keep it this way if IssuesDetails is expected to have a table in the future

Reply:
the plan is to add the tables for builds/tests that have this issue, see #705